client/src/components/MultiJurisdictionAudit/AASetup/Contests/_mocks.ts

```diff
@@ -278,4 +278,79 @@ export const contestsInputMocks = {
   ],
 }
 
+export const hybridContestsInputMocks = {
```

Comment:
could these just be in the test cases where they are used? it would make it easier to read (so you don't have to go to a separate file)